apps/protocol-frontend/src/components/DeleteContributionDialog.tsx

```diff
@@ -20,6 +20,7 @@ const DeleteContributionDialog = (props: {
 
   const { onClose } = useDisclosure();
   const cancelRef = useRef<HTMLButtonElement>(null);
+  const [deleteing, setDeleting] = useState(false);
```

Comment:
minor typo here that may cause confusion later -- `deleting, setDeleting` -- extra 'e' added to `deleting`